src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1838,8 +1856,10 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
             pResult.kind = CORINFO_CALL_KIND.CORINFO_CALL;
             pResult._nullInstanceCheck = (uint)(((flags & CORINFO_CALLINFO_FLAGS.CORINFO_CALLINFO_CALLVIRT) != 0) ? 1 : 0);
 
-            // TODO: Generics
-            // pResult.contextHandle;
+            // TODO: Support Generics
+            //
+            // For now, set the contextHandle to zero to indicate Generics context is not supported.
+            pResult.contextHandle = (CORINFO_CONTEXT_STRUCT *)0;
```

Comment:
E.g. it will be problem for inlining (inlining is currently disabled, but we should be able to enable now that we have the JIT-EE interface exception handling wrapper).